python/dgl/heterograph.py

```diff
@@ -631,6 +633,52 @@ def get_etype_id(self, etype):
         if etid is None:
             raise DGLError('Edge type "{}" does not exist.'.format(etype))
         return etid
+    #################################################################
+    # Batching
+    #################################################################
+    @property
+    def batch_size(self):
+        return len(self.batch_num_nodes(self.ntypes[0]))
+
+    def batch_num_nodes(self, ntype=None):
```

Comment:
The behavior seems to be different from the one described in `batch.py`, where `batch_num_nodes` returns a dictionary in the case of multiple node types.